src/views/pages/Swap/Swap.js

```diff
@@ -733,37 +740,24 @@ const Swap = () => {
                   {mode === 'token' && (
                     <>
                       <Row className="mb-3">
-                        <Col xs="5">
-                          <div className="text-card">
-                            Input{' '}
-                            <i
-                              className="icon-small icon-info icon-dark ml-2"
-                              id="tooltipInput"
-                              role="button"
-                            />
-                            <UncontrolledTooltip
-                              placement="right"
-                              target="tooltipInput"
-                            >
-                              Your input amount.
-                            </UncontrolledTooltip>
-                          </div>
+                        <Col xs="auto">
+                          <div className="text-card">Input</div>
                         </Col>
-                        <Col xs="7" className="text-right">
+                        <Col className="text-right">
                           <div className="output-card">
                             {swapInput1?.value &&
-                              formatFromUnits(swapInput1?.value, 10)}{' '}
+                              formatFromUnits(swapInput1?.value, 6)}{' '}
```

Comment:
make the txnDetails values rounded to 6 decimals and make the Cols auto for the descriptor + the rest of the Row for the actual txnDetail Col (repeated as seen throughout all areas)